connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -94,10 +175,13 @@ export async function saveSuccessSyncActivity(
 
     const now = new Date();
 
+    const firstSuccessfulSyncTime = connector.firstSuccessfulSyncTime || now;
```

Comment:
This seems like something we need to handle on the slack size as well no?

Reply:
Added to the `syncStatus` function that are shared with slack (wasn't aware of those previously)
https://github.com/dust-tt/dust/pull/390/commits/77d5e2132601ef2995934f09210b180a2cc33d6e
cc @lasryaric